src/tools/web/elektrad/find_handler_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestGetFind(t *testing.T) {
+	keyNames := []string{
+		"user/elektrad/find/result1",
+		"user/elektrad/find/result2",
+	}
+
+	for _, keyName := range keyNames {
+		setupKey(t, keyName)
+	}
+
+	w := testGet(t, "/kdbFind/user/elektrad/find")
```

Comment:
use /tests/testname here, see doc/TESTING.md

avoid camelcase

Reply:
`/kdbFind` is the route - not the key